packages/@okta/i18n/src/properties/login.properties

```
@@ -815,7 +815,15 @@ deviceTrust.universalLink.fallback.getOktaMobile.title = Get Okta Mobile
 # {0} and {1} are beginning and closing HTML elements.
 deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStore{1}, {0}search{1} for {0}Okta Mobile{1} and tap on {0}GET{1} Okta Mobile. Once installed, sign in to Okta Mobile and follow the instructions to secure your device.
 
+# Fastpass
+core.auth.factor.signedNonce.error.invalidEnrollment=Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
+core.auth.factor.signedNonce.error.invalidFactor=Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
```

if this and above message is same.. we can just use one common key. No need to create two keys for duplicate strings.

---

Actually, I have copied it from [the core](https://github.com/okta/okta-core/blob/0901ed2c7503633ad66e26e597fac5ebfabfd2a2/resources/src/main/resources/messages-untranslated.properties#L960). Because that core can return those 2 fault codes.
Please advise is it strongly preferred to update the core to use a single error code?

---

You can add one of these keys and the value of the other to our remapping tool in [i18nTransformer](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/i18nTransformer.js#L56).

Better - consider renaming both of these i18n keys to make them more straightforward to consumers:

```suggestion
fastpass.signedNonce.error = Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
```

Then, in the `i18nTransformer` we can map both of the API keys to our new `fastpass.signedNonce.error` key.

---

+1 ^^

You can keep just one string in login.properties and map these different keys in `i18nTransformer`.. No need to change backend.

---

updated